src/eXpansion/Bundle/AdminChat/ChatCommand/AdminReturnCommand.php

```diff
@@ -27,14 +20,20 @@ public function execute($login, InputInterface $input)
     {
         $nickName = $this->playerStorage->getPlayerInfo($login)->getNickName();
         $group = $this->getGroupLabel($login);
+        try {
+            $return = $this->connection->{$this->functionName}();
 
-        $return = $this->connection->{$this->functionName}();
-
-        $this->chatNotification->sendMessage(
-            $this->chatMessage,
-            $this->isPublic ? null : $login,
-            ['%adminLevel%' => $group, '%admin%' => $nickName, '%return%' => $return]
-        );
-
+            $this->chatNotification->sendMessage(
+                $this->chatMessage,
+                $this->isPublic ? null : $login,
+                ['%adminLevel%' => $group, '%admin%' => $nickName, '%return%' => $return]
+            );
+        } catch (\Exception $e) {
+            $this->chatNotification->sendMessage(
```

Comment:
1. Would be good to log the exceptions
2. To catch only dedicated exceptions and not everything